Readmes rach java8 (#744)
Esri/arcgis-runtime-samples-android: java/time-based-query/README.md

# Time based query

Query data using a time extent. 

![Image of time-based query](time-based-query.png)

## Use case

This workflow can be used to return records that are between a specified start and end date. For example, records of Canada goose sightings over time could be queried to only show sightings during the winter migration time period.

## How to use the sample

Run the sample, and a subset of records will be displayed on the map.

## How it works

> 1. An instance of `ServiceFeatureTable` is created by passing a URL to the REST endpoint of a time-enabled service. Time-enabled services will have TimeInfo defined in the service description. This information is specified in ArcMap or ArcGIS Pro prior to publishing the service.

Reviewer: Maybe it's better to word this as an imperative?
```suggestion
1. Create an instance of `ServiceFeatureTable` by passing a URL to the REST endpoint of a time-enabled service. Time-enabled services will have TimeInfo defined in the service description. This information is specified in ArcMap or ArcGIS Pro prior to publishing the service.
```

> 2. Create an instance of `ServiceFeatureTable` in manual cache mode, so you can control how and when the feature table is populated with data.

Reviewer: I think this step would be better worded as "set the feature request mode of ..." as we have already created the service feature table.

> 2. Create an instance of `ServiceFeatureTable` in manual cache mode, so that the developer can control how and when the feature table is populated with data.

Reviewer 2: mixed up grammar here.
```suggestion
2. Create an instance of `ServiceFeatureTable` in manual cache mode, so you can control how and when the feature table is populated with data.
```
